purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,260 @@
+package com.revenuecat.purchases.customercenter
+
+import com.revenuecat.purchases.paywalls.PaywallColor
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+typealias RCColor = PaywallColor
+
+@Serializable
+internal class CustomerCenterRoot(
+    @SerialName("customer_center") val customerCenter: CustomerCenterConfigData,
+)
+
+@Serializable
+class CustomerCenterConfigData(
```

Comment:
Also note that the constructors here are public, so test data can be created more easily from within the `RevenueCatUI` module.